homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,28 @@
-#pragma once
-
-int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
-
-int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
+#pragma once
+
+int fibonacci_iterative(int sequence) {
+    int t1 = 0;
+    int t2 = 1;

```

Comment:
What are 't1' and `t2` ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...